extensions/extensions-docker/aissemble-policy-decision-point/src/main/resources/docker/Dockerfile

```diff
@@ -1,18 +1,18 @@
-ARG DOCKER_BASELINE_REPO_ID
-ARG VERSION_AISSEMBLE
-FROM ${DOCKER_BASELINE_REPO_ID}boozallen/aissemble-quarkus:${VERSION_AISSEMBLE}
+FROM registry.access.redhat.com/ubi9/openjdk-17-runtime:1.20
 
 LABEL org.opencontainers.image.source="https://github.com/boozallen/aissemble"
 
-USER root
-RUN microdnf install jq
-USER 1001
+#This is technically the default if nothing is set, but this ensures if we move it `run-java.sh` will still be able to find it
+ENV JAVA_APP_DIR="/deployments"
 
-RUN mkdir -p /deployments/krausening/
-COPY target/dockerbuild/*.jar /deployments/
+COPY --chown=default ./src/main/resources/truststore/aissemble-secure.jks $JAVA_APP_DIR/
+COPY --chown=default ./src/main/resources/krausening/base/aiops-security.properties $JAVA_APP_DIR/krausening/base/
```

Comment:
Q:  I can't remember if you said we were keeping certain property files with the old aiops-* prefix.   Is this one supposed to start with aissemble-* ?